### PR TITLE
Scan password via QR and use label from otpauth when adding TOTP via QR

### PR DESCRIPTION
**Pull request by eldad**
* TOTP QR Scan: parse label from `otpauth` and set account name
* Add "Scan Password" button using QR for regular passwords

Resolves https://github.com/yogeshpaliyal/KeyPass/issues/91

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -67,4 +67,5 @@
     <string name="alert_black_secret_key">Please enter secret key</string>
     <string name="alert_black_account_name">Please enter account name</string>
     <string name="add_totp">Add TOTP</string>
-</resources>
\ No newline at end of file
+    <string name="scan_password">Scan Password</string>
```
Comment (yogeshpaliyal) [inline app/src/main/res/values/strings.xml:70]: "scan_password" is not translated in "hi" (Hindi) or "zh" (Chinese)

**Review by yogeshpaliyal — CHANGES_REQUESTED** [review]

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (eldad) [reply app/src/main/res/values/strings.xml:70]: @yogeshpaliyal I can't speak neither :smile: Shall I just use the result from Google translate?

This is what Google translate gives me:
Hindi पासवर्ड स्कैन करें
Chinese 扫描密码

Comment (yogeshpaliyal) [reply app/src/main/res/values/strings.xml:70]: @eldad this works great, thanks for these valuable contribution

Comment (eldad) [reply app/src/main/res/values/strings.xml:70]: Added now

**Comment by yogeshpaliyal** [comment]
Thanks @eldad  ❤️❤️

**Comment by eldad** [comment]
Thanks @yogeshpaliyal for the quick feedback and merging! :star_struck:

**Pull request merged**